archey/entry.py

```diff
@@ -8,17 +8,21 @@
 class Entry(AbstractBaseClass):
     """Module base class"""
     @abstractmethod
-    def __init__(self, name=None, value=None):
+    def __init__(self, name=None, value=None, options=None):
         # Each entry will have always have the following attributes...
-        # `name` (key);
-        # `value` (value of entry as an appropriate object)
-        # ... which are `None` by default.
-        self.name = name
+        # `name`: key (defaults to the instantiated entry class name);
+        # `value`: value of entry as an appropriate object;
+        # `options`: configuration options *specific* to an entry instance;
+        self.name = name or self.__class__.__name__
         self.value = value
+        self.options = options or {}
 
-        # Propagates a reference to `Configuration` singleton to each inheriting class.
-        self._configuration = Configuration()
```

Comment:
Global `Configuration` will be now "hidden" for entries, only i18n and specific options will be available for them.